{"hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK","doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSAuthenticationError"]]},"sections":[],"metadata":{"modules":[{"name":"TSAuthenticationSDK"}],"roleHeading":"Case","externalID":"s:19TSAuthenticationSDK0A5ErrorO8internalyACs0C0_pSgcACmF","symbolKind":"case","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"`internal`","kind":"identifier"},{"kind":"text","text":"("},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"text":"?)","kind":"text"}],"role":"symbol","title":"TSAuthenticationError.internal(_:)"},"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/tsauthenticationsdk\/tsauthenticationerror\/internal(_:)"]}],"abstract":[{"text":"The authorization attempt failed for an unknown reason.","type":"text"}],"schemaVersion":{"patch":0,"minor":3,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"`internal`"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"kind":"text","text":"?)"}],"platforms":["iOS"],"languages":["swift"]}]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSAuthenticationError\/internal(_:)"},"references":{"doc://com.TransmitSecurity.TSAuthenticationSDK/documentation/TSAuthenticationSDK/TSAuthenticationError/internal(_:)":{"type":"topic","abstract":[{"text":"The authorization attempt failed for an unknown reason.","type":"text"}],"url":"\/documentation\/tsauthenticationsdk\/tsauthenticationerror\/internal(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"`internal`"},{"text":"(","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"kind":"text","text":"?)"}],"kind":"symbol","identifier":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSAuthenticationError\/internal(_:)","role":"symbol","title":"TSAuthenticationError.internal(_:)"},"doc://com.TransmitSecurity.TSAuthenticationSDK/documentation/TSAuthenticationSDK":{"type":"topic","kind":"symbol","identifier":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK","abstract":[],"role":"collection","url":"\/documentation\/tsauthenticationsdk","title":"TSAuthenticationSDK"},"doc://com.TransmitSecurity.TSAuthenticationSDK/documentation/TSAuthenticationSDK/TSAuthenticationError":{"kind":"symbol","identifier":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSAuthenticationError","title":"TSAuthenticationError","role":"symbol","type":"topic","url":"\/documentation\/tsauthenticationsdk\/tsauthenticationerror","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"TSAuthenticationError"}],"navigatorTitle":[{"text":"TSAuthenticationError","kind":"identifier"}],"abstract":[]}}}